### PR TITLE
Added create and fetch method for rate-pyth plugin

### LINKED ISSUE #39: Pyth Rate plugin support
- Create new Pyth rate plugin
- Fetch existing pyth rate plugin

Ref: #24

### PR DESCRIPTION
**Pull request by iamakshat01**
1. 
image
2. Closes #39

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -33,13 +34,16 @@ use {
...
 const VYPER_CORE_ID: &str = "vyPErCcGJKQQBeeQ59gXcWrDyU4vBrq8qQfacwmsAsp";
-const REDEEM_LOGIC_FORWARD: &str = "BrpV1re8MshA8qskKVxcEG8zXG3vf2uLX6myeTKAyhsK";
 const RATE_SWITCHBOARD: &str  = "2hGXiH1oEQwjCXRx8bNdHTi49ScZp7Mj2bxcjxtULKe1";
+const RATE_PYTH: &str = "3mxtC2cGVhHucUg4p58MVzVqUKLyiy1zWqRkRQdgUBPT";
 const OTC: &str = "8aHSkExY28qCvg4gnTLU7y1Ev6HnpJ1NxuWb9XtEesVt";
+const REDEEM_LOGIC_FORWARD: &str = "BrpV1re8MshA8qskKVxcEG8zXG3vf2uLX6myeTKAyhsK";
 const REDEEM_LOGIC_SETTLE_FORWARD: &str = "6vBg1GMtKj7EYDLWWt6tkHoDWLAAksNPbKWiXMic99qU";
 const REDEEM_LOGIC_VANILLA_OPTION: &str = "8fSeRtFseNrjdf8quE2YELhuzLkHV7WEGRPA9Jz8xEVe";
+const PYTH_ORACLE_DEVNET: &str = "J83w4HKfqxwcq3BEMMkPFSppX3gqekLyLJBexebFVkix";
+const PYTH_ORACLE_MAINNET: &str = "Gnt27xtC473ZT2Mw5u8wZ68Z3gULkSTb5DuxJy7eJotD";
```
Comment (giacomo-cc) [inline src/main.rs]: same as switchboard, rate accounts are an user input

File: src/main.rs [hunk 2]
```diff
@@ -33,13 +34,16 @@ use {
     }
 };
 
-
+const DEVNET: &str = "https://api.devnet.solana.com";
```
Comment (giacomo-cc) [inline src/main.rs]: can you provide some more meaningful name for this const? like `PUBLIC_DEVNET_RPC_ENDPOINT`

**Review by giacomo-cc — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 2 above)
Comment (iamakshat01) [reply src/main.rs]: Removing it.

File: src/main.rs (hunk 1 above)
Comment (iamakshat01) [reply src/main.rs]: Done
image

**Review by giacomo-cc — APPROVED** [review]
lgtm

**Pull request merged**